src/main/java/org/springframework/integration/aws/config/xml/AWSNamespaceHandler.java

```diff
@@ -27,11 +26,12 @@
  * @author Artem Bilan
  * @since 0.5
  */
-public class AWSNamespaceHandler extends AbstractIntegrationNamespaceHandler {
+public class AwsNamespaceHandler extends AbstractIntegrationNamespaceHandler {
```

Comment:
OK. Never mind.
Found the solution: http://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git